### DISCUSSION TITLE
Virginie-Sankara CD

### DISCUSSION THREAD

**Discussion by virginie-sankara** [gc-da11yn/gc-da11yn.github.io#672] [Members]
Bonjour/Hi !

I’m new to open-source and excited to contribute to this project! I became a developer with the ambition to make tech more accessible and inclusive, and I’m eager to bring that mindset here.

I’m a full-stack developer with a focus on frontend and background in digital sociology and communication. 
As a bilingual french native speaker, I’d love to help with language-related tasks or localization.

Could you point me to beginner-friendly issues or areas where I can be most helpful? Looking forward to learning and contributing!

**Comment by shawnthompson** [comment]
Hi @virginie-sankara, looks like you found a few issues and made pull request already. 

Thanks again for helping out. Looking forward to working with you more.
